migration.json

```diff
@@ -132,55 +132,55 @@
     },
     "dao": {
       "0.0.1-rc.16": {
-        "name": "Concomitant Opera singer",
+        "name": "Obtuse Sawfiler",
```

Comment:
Why do all these names change all the time?

Reply:
Because we made them random

Reply:
But you are regenerating them each time?

Reply:
apparently yes

Reply:
yes